docs/releases/2.1.txt

```diff
@@ -201,6 +201,8 @@ Models
 
 * A ``BinaryField`` may now be set to ``editable=True`` if you wish to include
   it in model forms.
+* The new :meth:`.QuerySet.explain` method allows displaying the execution
```

Comment:
Please add a blank line between the points.

Reply:
Done, thanks!